tmp/README.md

```diff
@@ -0,0 +1,148 @@
+# Aplikacja ProteGO Safe
```

Comment:
To chyba skomitowaliście przez przypadek. Polecam dodanie katalogu tmp do .gitignore.